pages/contact.js

```diff
@@ -1,7 +1,102 @@
-import React from 'react';
+import { IoMdMail }  from 'react-icons/io';
+import { AiFillPhone } from "react-icons/ai";
+import { VscChevronDown } from "react-icons/vsc";
+import Link from 'next/link'
+
 
 const contact = () => {
-	return <div>contact</div>;
+	return (
+		<div className='bg-white mb-20'>
+			<div className='w-full p-10 bg-bottom bg-cover bg-[url("/images/contact-bg.jpg")]'>
+				<h1 className='neuton flex justify-center items-center text-white font-[700] text-[64px]' >Contact</h1>
+			</div>
+			<div className='flex justify-between text-black '>
```

Comment:
create div that wrap the flex div with all the collapses, and put "contianer mx-auto" in the className